aries-site/src/examples/components/tag/CreateTag/CreateTag.js

```diff
@@ -0,0 +1,190 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Form,
+  FormField,
+  Select,
+  Tag,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { tagData } from '../data';
+import {
+  AppContainer,
+  getRegExp,
+  TagPageHeader,
+  prefix,
+  TagResults,
+  updateCreateOption,
+} from '.';
+
+const defaultNameOptions = [];
+const defaultValueOptions = [];
+const defaultName = '';
+const defaultValue = '';
+const defaultTags = [];
+
+Object.entries(tagData).forEach(([, value]) => {
```

Comment:
Per my data structure comments, I think we will need/want to:
- [ ] validate the type of each entry
- [ ] apply a sort to alphabetize (can't assume data will be structured neatly)